### PR TITLE
containerization of is-website-vulnerable and README updated

### PR DESCRIPTION
**Pull request by darkwizard242**
## Description

This PR contains a Dockerfile for containerization of is-website-vulnerable that can be consumed under dockerhub account of @lirantal.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Related Issue

#52 

## Motivation and Context

For one-off isolated runs of is-website-vulnerable using Docker containers. 

## How Has This Been Tested?

Tested locally by building the Docker Image.
Performing Docker runs of the image using default value for ENV SCAN_URL as well as custom values specified during docker runs.

## Checklist:

- [x] I have updated the documentation (if required).
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I added a picture of a cute animal cause it's fun

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,47 @@
-<p align="center"><h1 align="center">
```
Comment (lirantal) [inline README.md]: @darkwizard242 it looks like you deleted some things that shouldn't have been deleted
can you update the PR to restore the original READMe format and just add the Docker section that you have?

**Comment by lirantal** [comment]
@darkwizard242 
1. I left a comment about updating the README
2. We indeed already use GitHub Actions, and how about we use Snyk to test the container image (https://snyk.io/test/docker/alpine:latest) ? If you'd like to also PR the actions file I'm happy to merge it.

File: README.md (hunk 1 above)
Comment (darkwizard242) [reply README.md]: @lirantal  - Ah sorry, that seems to have been due to the Markdown plugin on the Atom text editor I was using.  I will add correct the README and push the changes to the PR now.

Comment (lirantal) [reply README.md]: Cool thanks. I expected it was probably some auto-formatter :)

Comment (darkwizard242) [reply README.md]: Yeah, that's my bad. Sorry about that. I used vim this time lol.

Corrected and pushed the README back now 👍

**Comment by lirantal** [comment]
Looks great, thank you!

**Comment by darkwizard242** [comment]
> @darkwizard242
> 
> 1. I left a comment about updating the README
> 2. We indeed already use GitHub Actions, and how about we use Snyk to test the container image (https://snyk.io/test/docker/alpine:latest) ? If you'd like to also PR the actions file I'm happy to merge it.

@lirantal  - Yup, I don't mind doing that. Happy to contribute. I may not get a chance to get to it until the weekend. If it's okay with you, I would like to create a separate PR to update the GitHub Actions workflow to include:

1. Build Image from Docker file
2. Vulnerability Scanning of Container via Snyk
3. Push the built container image to your DockerHub Account (https://hub.docker.com/u/lirantal/)


In the meantime, could you create a Repository in your DockerHub account so that the changes I introduce can push the docker image built to your DockerHub account to a DockerHub repo like: `is-website-vulnerable` 

Once you do that, there will only be two things left:
1. Generate a Token on your DockerHub account for that new Docker Repository and add that Token into is-website-vulnerable GitHub repository's secrets (documentation: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) as something like `TOKEN`
2. Create another secret in is-website-vulnerable GitHub repository's secrets (https://github.com/lirantal/is-website-vulnerable/settings/secrets) as `USER` with value `lirantal`

I could utilize reference them in the Github Action workflow as mentioned in documentation so that the image build is pushed to the appropriate repo. 

If it all sounds good to you, then please feel free to create another issue and assign it to me. 👍

**Comment by lirantal** [comment]
Sounds wonderful 🎉
I want to stress that there is _no urgency_ in working on a new PR to add this functionality so I kindly ask that you use the weekend time for yourself or family ❤️

* New Docker Hub repo created: https://hub.docker.com/repository/docker/lirantal/is-website-vulnerable
* I added the Docker Hub token as a secret available to a GitHub Action as `DOCKERHUB_TOKEN` as well as a `USER` set to lirantal

All sounds good with me and thank you so much again for working on this 🤗

**Pull request merged**